### PR TITLE
[grafana] add the option to set resources for the test framework container

### PR DESCRIPTION
**Pull request by dbluxo**

### CODE REVIEW

**Review by zanhsieh — APPROVED** [review]

**Comment by dbluxo** [comment]
@zanhsieh Could you please merge it? 🙏

**Comment by zanhsieh** [comment]
@dbluxo I can't merge this PR alone. It required two individual approvals. 
cc @Xtigyro @zalegrala

**Review by zalegrala — APPROVED** [review]

**Comment by zalegrala** [comment]
Looks like we need a version bump on this PR, but otherwise okay.

**Pull request merged**
